sections/security/bcryptpasswords.md

```diff
@@ -0,0 +1,28 @@
+# Avoid using the Node.js Crypto library for passwords, use Bcrypt
+
+### One Paragraph Explainer
+When storing user passwords, using an adaptive hashing algorithm such as bcrypt, offered by the [bcrypt npm module](https://www.npmjs.com/package/bcrypt) is recommended as opposed to using the native Node.js Crypto module. `Math.random()` should also never be used for as part of any password or token generation due to it's predictability.
+
```

Comment:
@js-kyle I believe that many readers won't easily grasp the meaning of 'number of rounds can be specified'

Maybe a short explanation, highly friendly and abstracted, can make the explanation clearer to readers who are not familiar with bcrypt idea?

Reply:
Yep, agree with you there. I'll have a think about the best way to do this

Reply:
Maybe explain that using standard crypto algorithms an attacker can try many combinations very quickly, Bcrypt was born to address this and it's slow by design to limit the number of attempts/sec